### DISCUSSION TITLE
Removing background noise/music from vocals

### DISCUSSION THREAD

**Discussion by HiimAndrew123** [Anjok07/ultimatevocalremovergui#1150] [Q&A]
How do i remove background noises or music from vocals? Which models should i use?

**Comment by DAffAttack** [comment]
Was helpful for me:
https://www.youtube.com/watch?v=ykOKwz3eRUQ&t=23s

**Reply by ACEXAN** [reply]
Great tut, but he explains how to remove vocals from a track and refers this forum for more information 😆
